lte/gateway/c/core/oai/include/s11_messages_types.h

```diff
@@ -1247,7 +1247,7 @@ typedef struct itti_s11_release_access_bearers_request_s {
   // Private Extension Private Extension ///< optional
   /* GTPv2-C specific parameters */
   void* trxn;
-  edns_peer_ip_t edns_peer_ip;
+  //edns_peer_ip_t edns_peer_ip;
```

Comment:
any reason to keep this around if not used?

Reply:
As per the spec Release Access Bearer Req contains only the name of the originating node. But in magma AGW,  edns_peer_ip_t is included which is updated from the PDN context. edns_peer_ip_t is not used anywhere in sgw task. So I removed it from Release Access Bearer Req. But looks like OAI is using this as OAI test failed on CI. @lionelgo @Vyas20 can you confirm if this field is needed?